### DISCUSSION TITLE
👨‍🌾🧑‍🌾 Multi-Player Turn-Based Adventures

### DISCUSSION THREAD

**Discussion by GoliathRulz** [sunflower-land/sunflower-land#2049] [Ideas]
Bumpkins are social creatures not prone to live alone on their own islands by themselves.  They like adventure and getting together with other bumpkins to have a good time, and in this case adventure.

So, I would like to propose for consideration....

Bumpkin Team Quests:

Take your bumpkin on a team-based turn-based adventure with your friends, up to 3.  You and your friends can adventure to quests on the ever popular Treasure Island, or other islands, and venture into the dungeons below to seek out treasures of untold wealth (or maybe some not so much... but still fun stuff).  The adventures would be dungeon based and involve challenging puzzles; potential battles; and maybe even labyrinths.   The end goal of these is to ultimately beat the board; win the prize at the end and get bumpkin experience that can boost your bumpkin's skill levels (maybe a new skill line set).

You can craft items that would go into the quests to help you:

Food Items... any food item can be brought and be used for health regeneration;
Smoothies ... in this case smoothies can be brought, but they would provide a specific skill boost for X number of turns in the dungeon after being consumed; and
Weapons ... yes, weapons will be needed to protect you and your friends.  These can be crafted at the Blacksmith before going out.

Weaponsmith Station / Armorer:

Add a station on the island that can be built with resources that will allow for advanced weapons and armor to be crafted.  These are permanent and can be of different quality / grades.  They would wearable on your bumpkin at bumpkin.io and could be sold as an NFT if desired.  This allows for crafters to make items that could be made; extracted and sold on the open market if they don't wish to adventure.

Monsters:

Well all I can think of is bumpkins are afraid of cave trolls; bats and the other creepy crawlies that are in the caves commonly found on the islands.  This said, venture into the wrong island cave and you may randomly find yourself fighting the cave giant, which will take a lot to be successful.

**Comment by Nitr0z** [comment]
Propose different levels for each dungeon in order to reward those who don't reach the end (the further you go, better are the rewards) or different dungeons depending on the players' stuff/level. 

🗺️ 10 floors per dungeon with different dungeons ? on different maps ?

✔️ Levels : 
Lvl 10 to 20 : Treasure Island
Lvl 20 to 30 : Stone Haven
Lvl 30 to 40 : Sunflorea
Lvl 40 to 50 : ?
Lvl 50+ : Snow Kingdom

✨ Have a "portal" of dungeons, (like the nether in minecraft) and in this portal you can choose the dungeon from a menu

**Reply by GoliathRulz** [reply]
I like that idea.  Lines up with the intent of the islands.
